### PR TITLE
Update: Navi visualization report styles

### PR DESCRIPTION
**Pull request by jkusa**
Resolves #

## Description

This PR aligns the visualization toggle styles with the colors and styles with the rest of the app

## Proposed Changes

- Update visualization toggle styles

## Screenshots

Before:
Screen Shot 2019-09-04 at 12 51 39 PM

After:
Screen Shot 2019-09-04 at 12 48 42 PM

- Note: the purple buttons in the dummy app need to go

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/reports/app/styles/navi-reports/components/visualization-toggle.less [hunk 1]
```diff
@@ -7,21 +7,25 @@
...
-    padding: 4px 10px;
+    padding: 6px 10px;
     font-size: @font-size-mid;
     margin-right: -1px;
     cursor: pointer;
 
     &--is-active {
-      background-color: @navi-purple-lightest;
       pointer-events: none;
+      background: @denali-active-blue;
```
Comment (mattgraham1995) [inline packages/reports/app/styles/navi-reports/components/visualization-toggle.less]: 🔤

File: packages/core/app/styles/navi-core/common/buttons.less [hunk 2]
```diff
@@ -19,39 +18,31 @@
 }
 
 .btn-primary {
-  background-color: @navi-form-purple;
-  border-color: @navi-purple-border;
-  border-width: 1px;
+  background: linear-gradient(to right, #3697f2, #3570f4);
```
Comment (balajimanoharan) [inline packages/core/app/styles/navi-core/common/buttons.less]: Do we want vars for these colors?

File: packages/reports/app/styles/navi-reports/components/filter-builder.less [hunk 3]
```diff
@@ -3,15 +3,25 @@
...
+.filter-builder-dimension {
   &__subject {
     display: flex;
   }
 
   &__operator-dropdown {
+    .ember-power-select-options {
+      background: @navi-white;
+      color: @navi-gray-800;
+    }
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/filter-builder.less]: Can we remove this and use the base style?

File: packages/reports/app/styles/navi-reports/components/filter-builder.less [hunk 4]
```diff
@@ -3,15 +3,25 @@
...
     .ember-power-select-option {
       font-size: @font-size-mid-base;
       padding-left: 8px;
     }
   }
+
+  .ember-power-select-multiple-option {
+    background-color: @denali-active-blue;
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/filter-builder.less]: And this?

File: packages/reports/app/styles/navi-reports/components/filter-collection.less [hunk 5]
```diff
@@ -96,6 +96,10 @@
     }
   }
 
+  //.filter-values--dimension-select__dropdown {
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/filter-collection.less]: Remove

**Comment by kevinhinterlong** [comment]
new report screen

**Pull request merged**